app/src/main/res/layout/fragment_login.xml

```diff
@@ -60,16 +60,13 @@
             android:orientation="vertical"
             android:padding="@dimen/login_layout_padding">
 
-            <TextView
-                android:id="@+id/tvLoginInfo"
+            <ImageView
+                android:id="@+id/myExperimentIcon"
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+                android:layout_height="@dimen/myExperimentLogoHeight"
                 android:layout_gravity="center"
-                android:background="@color/grey_background"
-                android:padding="@dimen/login_info_padding"
-                android:text="@string/logininfo"
-                android:textSize="@dimen/login_text_size"
-                android:textStyle="normal|bold" />
+                android:src="@drawable/myexlogo"
```

Comment:
@Hiteshgautam01 I wonder that this image might be small on large devices like android tablet. Can you please do a quick check that this looks good at the too large or too small devices?